### ISSUE TITLE
Search windows is overlapping the menu

### ISSUE THREAD

**Issue by marceloavilaoliveira** [oracle/opengrok#2871]
- OpenGrok 1.2.23
- OpenJDK 1.8.0_212
- Ubuntu 18.04
- Tomcat
- Chrome & Firefox

This is the correct way OpenGrok shows the results:
 
![OpenGrok-1](https://user-images.githubusercontent.com/20242819/61637031-2beee500-ac6d-11e9-95a0-fc7e9464d09e.png)

But the search window overlaps the menu if something is added to data/header_include file:

![OpenGrok-2](https://user-images.githubusercontent.com/20242819/61637033-2beee500-ac6d-11e9-8fb3-6598a5ae0d1a.png)

The same happens if the xref line is too long and/or the window is too short:

![OpenGrok-3](https://user-images.githubusercontent.com/20242819/61637034-2c877b80-ac6d-11e9-9664-3515cf328f52.png)

**Comment by vladak** [comment]
The `header_include` part is a dup of #2841 - the workaround is to avoid putting too high elements in the include.

The long xref is the biggest problem I think.

**Comment by marceloavilaoliveira** [comment]
What do you mean by "too high elements" in the include? I got the same issue if I simply put the following in the header_include:

```
<a href="https://opengrok.cpqd.com.br">
    <img src="/images/opengrok.png" align="center"/>
</a>
```

Could you point to a header_include example which works?

**Comment by vladak** [comment]
single line of text works for me.

**Comment by marceloavilaoliveira** [comment]
Yes, just a single line works. This is not enough for a header :-(

**Comment by tony-zhao** [comment]
Following change can resolve the issue:

diff --git a/opengrok-web/src/main/webapp/default/style.css b/opengrok-web/src/main/webapp/default/style.css
index e66842c..8a28a51 100644
--- a/opengrok-web/src/main/webapp/default/style.css
+++ b/opengrok-web/src/main/webapp/default/style.css
@@ -276,7 +276,7 @@ html.diff #whole_header {
     position: fixed;
     top: 0;
     background: #ffffff;
-    height: 70px;
+    margin-bottom: 5px;
     width: 100%;
 }
 
@@ -473,7 +473,7 @@ html.xref #content,
 html.diff #content,
 html.more #content,
 html.history #content{
-    margin-top: 70px;
+    display: none;
 }
 
 /* *** help page *** */
diff --git a/opengrok-web/src/main/webapp/js/utils-0.0.31.js b/opengrok-web/src/main/webapp/js/utils-0.0.31.js
index 4a17bd8..440ff9b 100644
--- a/opengrok-web/src/main/webapp/js/utils-0.0.31.js
+++ b/opengrok-web/src/main/webapp/js/utils-0.0.31.js
@@ -2429,3 +2429,9 @@ function getCookie(cname) {
     }
     return "";
 }
+
+$(document).ready(function() {
+    whole_header_bottom = $('#whole_header').position().top + $('#whole_header').outerHeight(true);
+    $('#content').css('margin-top', whole_header_bottom);
+    $('#content').show();
+});

**Comment by marceloavilaoliveira** [comment]
> Following change can resolve the issue:

I made these changes up to the 1.2.25 version but the result was not good. The overlap issue wasn't fixed and the URL line anchors in the search results aren't working anymore.

For example, if we click on the link below

https://OPENGROK-SERVER/source/xref/PROJECT/PATH/FILENAME#28

The browser doesn't go to the line 28 of the filename.
